docs/developer/partition_api.rst

```diff
@@ -13,7 +13,7 @@ You can pass the partitions having needed IPs to your function. It can help with
 
 Partition API implementations
 -----------------------------
-By default, a Modin Dataframe stores underlying partitions as ``pandas.DataFrame`` objects.
+By default, a Core Modin Dataframe stores underlying partitions as ``pandas.DataFrame`` objects.
```

Comment:
same as above

Reply:
added a direct link to the high-level dataframe